src/test/a11y/tests/system-admin-routes.test.ts

```diff
@@ -125,4 +125,46 @@ describe('Accessibility - System Admin Routes', () => {
             testAccessibility(route.path, route.parameter);
         });
     });
+
+    describe('Page with Errors', () => {
+        describe('Create System Admin Account Page', () => {
+            const url = '/create-system-admin-account';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
+            });
+        });
+
+        describe('Reference Data upload Page', () => {
+            const url = '/reference-data-upload';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
+            });
+        });
+
+        describe('Delete Court Reference Data Confirmation Page', () => {
+            const url = '/delete-court-reference-data-confirmation';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
+            });
+        });
+
+        describe('Bulk Create Media Accounts Page', () => {
+            const url = '/bulk-create-media-accounts';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code